src/GraphQL.Tests/Execution/VariablesTests.cs

```diff
@@ -109,7 +109,7 @@ public void executes_with_complex_input()
               fieldWithObjectInput(input: {a: ""foo"", b: [""bar""], c: ""baz""})
             }
             ";
-            var expected = "{ \"fieldWithObjectInput\": \"{\\\"a\\\":\\\"foo\\\",\\\"b\\\":[\\\"bar\\\"],\\\"c\\\":\\\"baz\\\"}\" }";
+            var expected = "{ \"fieldWithObjectInput\": \"{\\\"a\\\":\\\"foo\\\",\\\"b\\\":[\\\"bar\\\"],\\\"c\\\":\\\"baz\\\",\\\"d\\\":null}\" }";
```

Comment:
Ah, okay. Let me know. If my changes has modified the expected behaviour here, I can fix that.